features/security/validate_response_types.feature

```diff
@@ -26,13 +26,13 @@ Feature: Validate response types
 
   Scenario: Requesting an invalid format in the URL should throw an error
     And I send a "GET" request to "/dummies/1.invalid"
-    Then the response status code should be 404
+    Then the response status code should be 406
```

Comment:
I think status 406 only applies to content negotiation using `Accept*` headers. This one should remain a 404 IMO.

Reply:
I think you're right, and this is why header fields should be preferred over extensions

Reply:
The extension support is just intended for debug/send in Slack/etc. Not for production usage.

Reply:
💡 good to know